Harry Belafonte Issues A Frightening Warning About America’s Future.txt

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
is this file required ?
what is this file. I think you have saved some article and just uploaded

Reply:
I don't know how that came. No, it's not required.
color_print doesn't work in IDLE. It will probably work in ubuntu shell. I don't have ubuntu so I can't test it.
And you can remove .pyc files.